test/helpers/utils.js

```diff
@@ -32,13 +32,81 @@ export function daysFromNow(days) {
   date.setDate(date.getDate() + days);
   return date;
 }
-  
+
 export function daysFromDate(date, days) {
   let newDate = new Date();
-  newDate.setDate(date.getDate() + days)
+  newDate.setDate(date.getDate() + days);
   return newDate;
 }
-  
+
 export function dateInSeconds(date) {
   return Math.floor(date.getTime() / 1000);
-}
\ No newline at end of file
+}
+
+/**
+ * Returns a mock FundingCyleMetadata packed into a BigNumber
+ * @summary Should mirror the bit logic in JBFundingCycleMetadataResolver.sol.
+ * @param {custom obj} e.g. packFundingCycleMetadata({ reservedRate: 3500, pausePay: 1})
+ * @return {ethers.BigNumber}
+ * @note Passing in an empty obj will use default values below
+ */
```

Comment:
While you're in the neighborhood would you mind adding a similar doc to other fns? :)

Reply:
Community service